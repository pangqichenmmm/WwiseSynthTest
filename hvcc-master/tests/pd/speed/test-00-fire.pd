#N canvas 300 193 409 182 10;
#X obj 300 57 hip~ 1000;
#X obj 105 57 bp~ 1200 0.6;
#X obj 198 59 bp~ 2600 0.4;
#X obj 18 57 bp~ 600 0.2;
#X obj 152 118 dac~;
#N canvas 357 401 450 300 firegen 0;
#N canvas 0 22 242 473 crackles 0;
#X obj 64 394 *~;
#X obj 77 104 env~;
#X obj 77 80 lop~ 1;
#X msg 128 170 bang;
#X obj 77 127 moses 50;
#X obj 128 148 moses 51;
#X obj 128 193 random 30;
#X obj 128 237 line~;
#X obj 128 265 *~;
#X obj 96 325 + 1500;
#X obj 64 349 bp~ 4000 1;
#X obj 95 303 * 500;
#X obj 64 419 outlet~;
#X obj 63 52 inlet~;
#X msg 128 216 1 \, 0 15;
#X connect 0 0 12 0;
#X connect 1 0 4 0;
#X connect 2 0 1 0;
#X connect 3 0 6 0;
#X connect 4 1 5 0;
#X connect 5 0 3 0;
#X connect 6 0 14 0;
#X connect 6 0 11 0;
#X connect 7 0 8 0;
#X connect 7 0 8 1;
#X connect 8 0 0 1;
#X connect 9 0 10 1;
#X connect 10 0 0 0;
#X connect 11 0 9 0;
#X connect 13 0 2 0;
#X connect 13 0 10 0;
#X connect 14 0 7 0;
#X restore 55 97 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 264 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 105 32 pd firegen;
#N canvas 27 401 424 180 firegen 0;
#N canvas 0 22 242 473 crackles 0;
#X obj 64 394 *~;
#X obj 77 104 env~;
#X obj 77 80 lop~ 1;
#X msg 128 170 bang;
#X obj 77 127 moses 50;
#X obj 128 148 moses 51;
#X obj 128 193 random 30;
#X obj 128 237 line~;
#X obj 128 265 *~;
#X obj 96 325 + 1500;
#X obj 64 349 bp~ 4000 1;
#X obj 95 303 * 500;
#X obj 64 419 outlet~;
#X obj 63 52 inlet~;
#X msg 128 216 1 \, 0 15;
#X connect 0 0 12 0;
#X connect 1 0 4 0;
#X connect 2 0 1 0;
#X connect 3 0 6 0;
#X connect 4 1 5 0;
#X connect 5 0 3 0;
#X connect 6 0 14 0;
#X connect 6 0 11 0;
#X connect 7 0 8 0;
#X connect 7 0 8 1;
#X connect 8 0 0 1;
#X connect 9 0 10 1;
#X connect 10 0 0 0;
#X connect 11 0 9 0;
#X connect 13 0 2 0;
#X connect 13 0 10 0;
#X connect 14 0 7 0;
#X restore 60 77 pd crackles;
#N canvas 445 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 152 77 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 249 78 pd lapping;
#X obj 151 42 noise~;
#X text 206 41 factor common noise;
#X text 308 110 mix levels;
#X obj 154 135 outlet~;
#X obj 60 104 *~ 0.2;
#X obj 249 104 *~ 0.6;
#X obj 153 103 *~ 0.3;
#X text 147 11 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 198 32 pd firegen;
#N canvas 355 443 450 300 firegen 0;
#N canvas 0 22 242 473 crackles 0;
#X obj 64 394 *~;
#X obj 77 104 env~;
#X obj 77 80 lop~ 1;
#X msg 128 170 bang;
#X obj 77 127 moses 50;
#X obj 128 148 moses 51;
#X obj 128 193 random 30;
#X obj 128 237 line~;
#X obj 128 265 *~;
#X obj 96 325 + 1500;
#X obj 64 349 bp~ 4000 1;
#X obj 96 299 * 500;
#X obj 64 419 outlet~;
#X obj 63 52 inlet~;
#X msg 128 216 1 \, 0 15;
#X connect 0 0 12 0;
#X connect 1 0 4 0;
#X connect 2 0 1 0;
#X connect 3 0 6 0;
#X connect 4 1 5 0;
#X connect 5 0 3 0;
#X connect 6 0 14 0;
#X connect 6 0 11 0;
#X connect 7 0 8 0;
#X connect 7 0 8 1;
#X connect 8 0 0 1;
#X connect 9 0 10 1;
#X connect 10 0 0 0;
#X connect 11 0 9 0;
#X connect 13 0 2 0;
#X connect 13 0 10 0;
#X connect 14 0 7 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 300 32 pd firegen;
#N canvas 283 483 450 300 firegen 0;
#N canvas 8 22 242 473 crackles 0;
#X obj 64 394 *~;
#X obj 77 104 env~;
#X obj 77 80 lop~ 1;
#X msg 128 170 bang;
#X obj 77 127 moses 50;
#X obj 128 148 moses 51;
#X obj 128 193 random 30;
#X obj 128 237 line~;
#X obj 128 265 *~;
#X obj 96 325 + 1500;
#X obj 64 349 bp~ 4000 1;
#X obj 95 303 * 500;
#X obj 64 419 outlet~;
#X obj 63 52 inlet~;
#X msg 128 216 1 \, 0 15;
#X connect 0 0 12 0;
#X connect 1 0 4 0;
#X connect 2 0 1 0;
#X connect 3 0 6 0;
#X connect 4 1 5 0;
#X connect 5 0 3 0;
#X connect 6 0 14 0;
#X connect 6 0 11 0;
#X connect 7 0 8 0;
#X connect 7 0 8 1;
#X connect 8 0 0 1;
#X connect 9 0 10 1;
#X connect 10 0 0 0;
#X connect 11 0 9 0;
#X connect 13 0 2 0;
#X connect 13 0 10 0;
#X connect 14 0 7 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 18 32 pd firegen;
#X obj 152 93 *~ 0.2;
#X connect 0 0 9 0;
#X connect 1 0 9 0;
#X connect 2 0 9 0;
#X connect 3 0 9 0;
#X connect 5 0 1 0;
#X connect 6 0 2 0;
#X connect 7 0 0 0;
#X connect 8 0 3 0;
#X connect 9 0 4 0;
#X connect 9 0 4 1;
