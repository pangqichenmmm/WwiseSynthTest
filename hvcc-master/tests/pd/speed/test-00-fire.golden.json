{
  "samplerate": 48000.0,
  "blocksize": 512,
  "numIterations": 2000000,
  "usPerBlock": {
    "HV_SIMD_SSE": 31.765863,
    "HV_SIMD_AVX": 33.890763
  }
}
